src/lib/es2024.collection.d.ts

```diff
@@ -0,0 +1,11 @@
+interface MapConstructor {
+    /**
+     * Groups members of an iterable according to the return value of the passed callback.
+     * @param items An iterable.
+     * @param keySelector A callback which will be invoked for each item in items.
+     */
+    groupBy<K, T>(
+        items: Iterable<T>,

```

Comment:
Given #58243 has merged, this should probably be `Iterable<T, unknown, undefined>`.

```suggestion
        items: Iterable<T, unknown, undefined>,
```